Makefile

```diff
@@ -4,7 +4,7 @@ GO_LDFLAGS_STATIC=-ldflags "-s -w -extldflags -static"
 default: dev
 
 dev:
-	go run ./cmd/gaia/main.go -homepath=${PWD}/tmp -dev=true -poll=true
+	go run ./cmd/gaia/main.go -homepath=${PWD}/tmp -dev=true
```

Comment:
I guess we already talked about this. I think we can leave it in. What do you think? 🤗

Reply:
I will just merge it in now. We can add it later again if decision has been made 🤗